src/gluonnlp/data/transforms.py

```diff
@@ -704,26 +718,26 @@ class BasicTokenizer():
     performs invalid character removal (e.g. control chars) and whitespace.
     tokenize CJK chars.
     splits punctuation on a piece of text.
-    strips accents and convert to lower case.(If lower_case is true)
+    strips accents and convert to lower case.(If do_lower_case is true)
 
     Parameters
     ----------
-    lower_case : bool, default True
+    do_lower_case : bool, default True
```

Comment:
Why change `lower_case` to `do_lower_case`，I also think that lower_case is more concise. https://github.com/dmlc/gluon-nlp/pull/464#discussion_r243149189

Reply:
Why don't we follow Python standard library and call it consistently `lower`? See https://docs.python.org/3/library/stdtypes.html#str.lower
Btw, lowercase can be both state and action. See eg. https://en.wiktionary.org/wiki/lowercase#English. lowercase is a valid verb.

Reply:
"lower" sounds good!

Reply:
It has been modified